src/test/java/org/fundacionjala/coding/marco/EANValidatorTest.java

```diff
@@ -0,0 +1,33 @@
+package org.fundacionjala.coding.marco;
+
+import org.junit.Test;
+
+
+import static junit.framework.TestCase.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * this is my test.
+ */
+public class EANValidatorTest {
```

Comment:
to increase the code coverage you should test when the checksum is 0.
for instance.
```java
 assertTrue(validator.validator("9783827317100"));
```